skyportal/handlers/api/observation_plan.py

```diff
@@ -0,0 +1,150 @@
+import jsonschema
+from marshmallow.exceptions import ValidationError
+
+from baselayer.app.access import auth_or_token
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    ObservationPlanRequest,
+    Group,
+    Allocation,
+)
+
+from ...models.schema import ObservationPlanPost
+
+
+class ObservationPlanRequestHandler(BaseHandler):
```

Comment:
Aren't we going to need a GET function at some point?

Reply:
Or is the idea that you can find the list of these requests by the `GcnEvent` getter?

Reply:
Yeah that's how we show them off on the front page. We do the same for FollowupRequests... no idea if that is best practice or not, just folliowing that.

Reply:
I think in general we prefer to load separate things using separate API calls. I would say a leave the functionality of getting all the localization etc along with the GCN object (because it is very convenient), but also make sure you have at least GET, POST, DELETE (if not PATCH as well) for every handler.